Code/Tools/LevelEdit/TGAToDXT.cpp

```diff
@@ -176,10 +140,13 @@ bool TGAToDXTClass::Convert (const char *inputpathname, const char *outputpathna
 ///////////////////////////////////////////////////////////////////////////////
 void TGAToDXTClass::Write (const char *outputpathname)
 {
+	if (!Buffer)
+		return;
```

Comment:
I think the code convention is to always use curly braces.
```suggestion
	if (!Buffer) {
		return;
	}
```